### PR TITLE
LPS-103362 Update product-navigation:personal-menu tag

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-103362](https://issues.liferay.com/browse/LPS-103362).<br><br>/cc @jbalsas

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/resources/META-INF/resources/control_menu/personal_menu_icon.jsp [hunk 1]
```diff
@@ -40,27 +41,9 @@
...
-						<aui:icon id="impersonate-user-icon" image="user" markupView="lexicon" />
-					</span>
-				</c:if>
-			</span>
-		</liferay-util:buffer>
-
 		<liferay-product-navigation:personal-menu
 			expanded="<%= true %>"
-			label="<%= userAvatar %>"
+			user="<%= user %>"
```
Comment (pei-jung) [inline modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/resources/META-INF/resources/control_menu/personal_menu_icon.jsp]: The element that triggers the personal menu needs to be customizable. If we're deprecating `label` here, we need an alternative way to set the custom trigger.

It's a feature requested here: https://issues.liferay.com/browse/LPS-90244

Comment (jbalsas) [reply modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/resources/META-INF/resources/control_menu/personal_menu_icon.jsp]: Hey @pei-jung, is this a real feature request? The way it's used right now it doesn't look like this feature needs the whole label to be customizable, just to add support for the options you want.

Comment (jbalsas) [reply modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/resources/META-INF/resources/control_menu/personal_menu_icon.jsp]: In any case, if this is a real feature request your team wants to offer and maintain, it should be as simple as allowing `label` to be passed down to the tag (as it is right now), and then using the value both in the `page.jsp` for the initial render as the value to `dangerouslySetInnerHTML` in the React component for the trigger.

Comment (pei-jung) [reply modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/resources/META-INF/resources/control_menu/personal_menu_icon.jsp]: FYI, it's part of the requirements for https://issues.liferay.com/browse/LPS-90313

**Comment by jbalsas** [comment]
Hey @pei-jung, @drewbrokke, could you please take a look at https://github.com/pei-jung/liferay-portal/pull/527?

**Comment by jbalsas** [comment]
ci:close

**Pull request closed without merging**